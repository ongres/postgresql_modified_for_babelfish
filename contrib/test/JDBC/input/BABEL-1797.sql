USE master
GO

create table [dbo].[t23]([id] int, [a] money, [b] datetime)
go

exec sp_describe_undeclared_parameters
N'INSERT INTO [dbo].[t23]([id],[a],[b]) values (@P1,@P2,@P3)'
go

-- cleanup
drop table [dbo].[t23];
go
