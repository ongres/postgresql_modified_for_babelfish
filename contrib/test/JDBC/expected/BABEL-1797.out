USE master
GO

create table [dbo].[t23]([id] int, [a] money, [b] datetime)
go

exec sp_describe_undeclared_parameters
N'INSERT INTO [dbo].[t23]([id],[a],[b]) values (@P1,@P2,@P3)'
go
~~START~~
int#!#varchar#!#int#!#nvarchar#!#smallint#!#tinyint#!#tinyint#!#int#!#varchar#!#varchar#!#varchar#!#nvarchar#!#int#!#varchar#!#varchar#!#varchar#!#bit#!#bit#!#bit#!#bit#!#bit#!#varchar#!#int#!#int
1#!#@p1#!#0#!##!#4#!#32#!#0#!#<NULL>#!##!##!##!##!#<NULL>#!##!##!##!#0#!#0#!#0#!#0#!#0#!##!#0#!#0
2#!#@p2#!#0#!##!#21#!#0#!#0#!#<NULL>#!##!##!##!##!#<NULL>#!##!##!##!#0#!#0#!#0#!#0#!#0#!##!#0#!#0
3#!#@p3#!#0#!##!#16#!#0#!#0#!#<NULL>#!##!##!##!##!#<NULL>#!##!##!##!#0#!#0#!#0#!#0#!#0#!##!#0#!#0
~~END~~


-- cleanup
drop table [dbo].[t23];
go
